From b80a7f928153c1ce4d281d03081a7e8692e602ba Mon Sep 17 00:00:00 2001
From: Sathesh Edara <sedara@marvell.com>
Date: Mon, 2 Jan 2023 03:15:08 -0800
Subject: [PATCH 133/955] net/octeon_ep: print packet mode and pkind

add printing at startup of the selected packet
mode (loop or nic), and the pkind associated
with it.

Signed-off-by: Sathesh Edara <sedara@marvell.com>
Change-Id: I2c164d4916dbf703b8b585416e565aa05f3dc6bc
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/dataplane/dpdk/+/93497
Tested-by: sa_ip-toolkits-Jenkins <sa_ip-toolkits-jenkins@marvell.com>
Reviewed-by: Veerasenareddy Burru <vburru@marvell.com>
---
 drivers/net/octeon_ep/otx_ep_ethdev.c | 11 ++++++++---
 1 file changed, 8 insertions(+), 3 deletions(-)

diff --git a/drivers/net/octeon_ep/otx_ep_ethdev.c b/drivers/net/octeon_ep/otx_ep_ethdev.c
index 36227dabd3bd7..39703d2b5fb5a 100644
--- a/drivers/net/octeon_ep/otx_ep_ethdev.c
+++ b/drivers/net/octeon_ep/otx_ep_ethdev.c
@@ -599,17 +599,22 @@ otx_ep_eth_dev_init(struct rte_eth_dev *eth_dev)
 	    otx_epvf->chip_id == PCI_DEVID_CN10KB_EP_NET_VF ||
 	    otx_epvf->chip_id == PCI_DEVID_CNF10KA_EP_NET_VF ||
 	    otx_epvf->chip_id == PCI_DEVID_CNF10KB_EP_NET_VF) {
-		if (otx_epvf->sdp_packet_mode == SDP_PACKET_MODE_NIC)
+		if (otx_epvf->sdp_packet_mode == SDP_PACKET_MODE_NIC) {
 			otx_epvf->pkind = SDP_OTX2_PKIND_FS24;
-		else
+			otx_ep_info("Using pkind %d for NIC packet mode.\n",
+				  otx_epvf->pkind);
+		} else {
 			otx_epvf->pkind = SDP_OTX2_PKIND_FS0;
+			otx_ep_info("Using pkind %d for LOOP packet mode.\n",
+				  otx_epvf->pkind);
+		}
 	} else if (otx_epvf->chip_id == PCI_DEVID_OCTEONTX_EP_VF) {
 		otx_epvf->pkind = SDP_PKIND;
+		otx_ep_info("Using pkind %d.\n", otx_epvf->pkind);
 	} else {
 		otx_ep_err("Invalid chip id\n");
 		return -EINVAL;
 	}
-	otx_ep_info("using pkind %d\n", otx_epvf->pkind);
 
 	return 0;
 }
-- 
2.25.1

